### PR TITLE
Add Vector -from-tasty benchmark

### LINKED ISSUE #3843: Add benchmarks to measure TASTY unpickling performance
It would be good to have benchmarks that compile with `-from-tasty` to be able to track our unpickling performance.

### PR DESCRIPTION
**Pull request by nicolasstucki**
To fix https://github.com/lampepfl/dotty/issues/3843

### CODE REVIEW

File: profiles/default.yml [hunk 1]
```diff
@@ -13,6 +13,12 @@ charts:
         - key: vector
           label: bootstrapped
 
+    - name: Vector -from-tasty
+      url: https://github.com/liufengyun/bench/blob/master/tests/Vector.scala
+      lines:
+        - key: vector-from-tasty
+          label: bootstrapped
+
```
Comment (liufengyun) [inline profiles/default.yml]: If you want to show them in the same chart, just add a new line under `name: Vector`

**Review by liufengyun — APPROVED** [review]
Otherwise, LGTM

**Comment by nicolasstucki** [comment]
@liufengyun could you run the benchmarks for this at least 1 month back. It would be interesting to see what happend when we switched from TASTy in the classfiles to TASTy files.

**Comment by liufengyun** [comment]
OK, I'll do that right now.

**Comment by liufengyun** [comment]
Scheduled jobs for every 3 PR merges since #4777  (https://github.com/lampepfl/dotty/commit/d28ba0691fcda6a08769de7bd0322dd2881721e3), which is merged on 9 July.

```
bin/run -p profiles/tasty.plan -f 4777 -s 2
```

```
fliu@lamppc37:/data/workspace/bench$ cat profiles/tasty.plan
set -e

set-key vector-from-tasty
source $PROG_HOME/tests/scripts/vector-from-tasty
```

**Pull request merged**